test/common/testlib.py

```diff
@@ -784,9 +785,22 @@ def enable_preload(self, package, *pages):
         self.write_file(path, '{ "preload": [%s]}' % ', '.join('"{0}"'.format(page) for page in pages))
 
     def setUp(self, restrict=True):
+        if os.getenv("MACHINE"):
+            # apply env variable together if MACHINE envvar is set
+            opts.address = os.getenv("MACHINE")
+            if os.getenv("DESTRUCTIVE") and not self.is_nondestructive():
+                print("run destructive test, be careful, may lead to upredictable state")
+            else:
+                raise unittest.SkipTest("Skip test what is not marked as nondestructive")
```

Comment:
Just "Skipping destructive test" should do; or at least s/what/that/

Reply:
fixed